spec/acceptance/exec_pwsh_spec.rb

```diff
@@ -507,6 +540,31 @@ def platform_string(host, windows, posix)
         its(:content) { should match /svchost/ }
       end
     end
+
+    context 'on POSIX platforms', :if => posix6_agents.count > 0 do
```

Comment:
Another one where I'd rather see us share a test and make it universal if possible.

Reply:
Done.